src/app/containers/MostRead/utilities/processMostRead.js

```diff
@@ -83,8 +80,8 @@ const mostReadItems = ({ data, numberOfItems }) => {
       if (href && title) {
         items.push(mostReadItemData);
       } else {
-        logMissingDataFields({
-          error: 'Most read item is missing title or link data fields',
+        logger.warn(MOST_READ_DATA_INCOMPLETE, {
+          message: 'Most read item is missing title or link data fields',
```

Comment:
Do you need to determine which field is empty, or does it matter? (I suppose the absense of either one is not good)

Reply:
I've updated this so that it includes the value of href and title because I think it would make it easier to find what record had a missing data value.